ddtrace/tracer/time_windows.go

```diff
@@ -37,3 +37,12 @@ var now func() int64 = func() func() int64 {
 		return highPrecisionNow
 	}
 }()
+
+var nowTime func() time.Time = func() func() time.Time {
+	if err := windows.LoadGetSystemTimePreciseAsFileTime(); err != nil {
+		log.Warn("Unable to load high precison timer, defaulting to time.Now()")
+		return func() time.Time { return time.Unix(0, lowPrecisionNow()) }
```

Comment:
Just curious here, how did you figure out that the timer returned by `LoadGetSystemTimePreciseAsFileTime()` returned a unix time? Looking at the windows api [docs](https://learn.microsoft.com/en-us/windows/win32/api/minwinbase/ns-minwinbase-filetime) seems to suggest it's something totally different? (Which might mean our high precision time was always wrong?)